src/main/java/com/google/api/codegen/ruby/RubyGapicContext.java

```diff
@@ -358,6 +361,54 @@ public DynamicLangApiMethodView getMethodView(Interface service, Method method)
     return methodTransformer.generateDynamicLangApiMethod(methodContext);
   }
 
+  public List<GrpcStubView> getStubs(Interface service) {
+    ModelTypeTable modelTypeTable =
+        new ModelTypeTable(
+            new RubyTypeTable(getApiConfig().getPackageName()),
+            new RubyModelTypeNameConverter(getApiConfig().getPackageName()));
+    SurfaceTransformerContext context =
+        SurfaceTransformerContext.create(
+            service,
+            getApiConfig(),
+            modelTypeTable,
+            new RubySurfaceNamer(getApiConfig().getPackageName()));
```

Comment:
Create a helper function to create the context for you, so it isn't duplicated between here and getMethodView()